.gitlint

```diff
@@ -63,8 +64,8 @@ regex=\[(GS|DD|WB|WF)\] (Fix|New|Update): [A-Z]+[a-zA-Z0-9- ]+
 # [body-max-line-length]
 # line-length=72
 
-# [body-min-length]
-# min-length=5
+[body-min-length]
+min-length=5
```

Comment:
any reason for min-length?

Reply:
The default body length is 20 characters (see https://jorisroovers.com/gitlint/rules/). In case a body is provided, we shouldn't be so strict regarding the character count, that's why I reduced it.